integration/config/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+shipyard_dir=".e2e-harness/workdir/.shipyard"
+mkdir -p "${shipyard_dir}"
+chmod -R 777 ".e2e-harness"
+kind get kubeconfig --name="kind" | sudo tee "${shipyard_dir}/config" > /dev/null
+
+
+shipyard_dir="/workdir/.shipyard"
+sudo mkdir -p "${shipyard_dir}"
+sudo chmod -R 777 "/workdir"
+kind get kubeconfig --name="kind" | sudo tee "${shipyard_dir}/config" > /dev/null
```

Comment:
I'd skip `sudo` here. The parent dir has already 777 and we want this file to be readable by the program executed with `go test`.

```suggestion
kind get kubeconfig --name="kind" > "${shipyard_dir}/config"
```